### PR TITLE
Autoinjector Embiggen

### PR DESCRIPTION
**Pull request by Geevies**
* Autoinjectors have been resprited, they are now slightly bigger.
* Special autoinjectors that spawn with reagents are properly labeled, which allows you to take the label off whenever.
* Autoinjectors have received some flavour text in its extended description.

Look at all these flavours!
image

Easier to see if the back is open, to pour reagents inside.
image

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by NonQueueingMatt** [comment]
These look honestly a bit too big. I would shrink them by a few pixels.

**Comment by Geevies** [comment]
i sent these to you on discord, but i'm putting it here for reviewers too, it looks a lot smaller when zoomed out
image
image

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -104,19 +104,26 @@
...
 	volume = 5
 	time = 0
 
 /obj/item/reagent_containers/hypospray/autoinjector/Initialize()
-	. =..()
-	icon_state = empty_state
-	item_state = empty_state
+	. = ..()
+	if(name_label)
+		name_unlabel = name
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/hypospray.dm:121]: this is just initial(name)

**Review by NonQueueingMatt — DISMISSED** [review]

File: code/modules/reagents/reagent_containers/hypospray.dm (hunk 1 above)
Comment (Geevies) [reply code/modules/reagents/reagent_containers/hypospray.dm:121]: this is setting the name for the remove label proc

**Review by JohnWildkins — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**